terraform/main.tf

```diff
@@ -13,7 +13,7 @@ terraform {
     region = "eu-west-2"
     key    = "state"
   }
-  required_version = "1.6.4"
+    required_version = ">= 1.5.0"
```

Comment:
You sure this works? I used 1.6.4 because it was required the for modules I used, though I'm struggling to find a source for that.
If so then you'll need to change `.terraform-version` as well.

Reply:
This works for me. I think locking this to the newest will cause unnecessary build failures. That `.terraform-version` can be to the latest version (as it is now) and shouldn't cause issues:
>Please note that we don't do semantic version range parsing but use first ever found version as the candidate for minimally required one. It is up to the user to keep the definition reasonable.